storage/wdpost_run.go

```diff
@@ -762,14 +763,32 @@ func (s *WindowPoStScheduler) setSender(ctx context.Context, msg *types.Message,
 	}
 	*msg = *gm
 
+	mm := *msg
+
+	mm.GasPremium, err = s.api.GasEstimateGasPremium(ctx, 5, msg.From, msg.GasLimit, types.TipSetKey{})
+	if err != nil {
+		log.Errorf("failed to estimate minimum gas premium: %+v", err)
+		mm.GasPremium = msg.GasPremium
+	}
+
+	mm.GasFeeCap, err = s.api.GasEstimateFeeCap(ctx, &mm, 2, types.EmptyTSK)
```

Comment:
This should for sure be more than 2, if you are swetting Premium for 5 block inclusion.

Reply:
Set it a bit higher, bust still pretty low - this is estimating the lowest fee we'll set on the message if we don't have enough funds in any account to set more. If we have enough funds, we'll use the default estimation from GasEstimateMessageGas above